### DISCUSSION TITLE
How to validate if Image is Monochrome Bitmap or not ?

### DISCUSSION THREAD

**Discussion by piyushmeshram2014** [SixLabors/ImageSharp#2182] [Q&A]
Hi Team,
I am looking for a way to validate an Image object against certain contraints which I have.Of all,one of them is to strictly have bitmap image as a monochrome. Is there a way to check the same ?

BasicallY i want a certain value from ImageSharp object which can give me value equal to (SixLabors.ImageSharp.Formats.Bmp.BmpBitsPerPixel.Pixel1)

**Comment by JimBobSquarePants** [comment]
You can use the metadata

https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Formats.Bmp.BmpMetadata.html#SixLabors_ImageSharp_Formats_Bmp_BmpMetadata_BitsPerPixel
